android/java/org/chromium/chrome/browser/vpn/adapters/VpnServerAdapter.java

```diff
@@ -44,6 +47,28 @@ public ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
         return new ViewHolder(listItem);
     }
 
+    private static enum CheckedState {
```

Comment:
A nested enum class is implicitly static. It's redundant since the enum is already static by nature. It's allowed though.